www/routes/docs/[...slug].tsx

```diff
@@ -140,6 +140,21 @@ function MobileSidebar(props: { path: string }) {
           </nav>
         </div>
       </div>
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+          var input = document.querySelector('#docs_sidebar');
+          if (input) {
+            input.addEventListener("change", function (ev) {
+              var value = ev.target.checked;
+              var main = document.querySelector("main");
+              if (!main) return;
+              main.classList.toggle("overflow-hidden");
```

Comment:
```suggestion
              main.classList.toggle("${tw`overflow-hidden`}");
```

Otherwise this class may not be generated if we happen to remove `overflow-hidden` from the other components that use it.